.env.sample

```
@@ -0,0 +1,2 @@
+BOT_TOKEN=""
+PREFIX=""
```

```suggestion
BOT_TOKEN=
PREFIX=
```
空白が含まれない変数はクォーテーションマーク必要無いです。